src/commons/BrowserUtils.ts

```diff
@@ -851,7 +856,7 @@ export namespace BrowserUtils {
    * For reporter log propose in order to log what keys were send
    * @param keysToSend key/keys of SpecialKeys type
    */
-  function getKeyNames(keysToSend: SpecialKeys | SpecialKeys[]): string {
+  function getKeyNames(keysToSend: SpecialKeys | SpecialKeys[] | string[]): string {
```

Comment:
does not looks that this function have been called with string[]

Reply:
obligated by rules of ts.